File: Examples/CSharp/Export/ExportImageToPSD.cs
using System.IO;

using Aspose.Imaging;

namespace Aspose.Imaging.Examples.Export
{
    public class ExportImageToPSD
    {
        public static void Main(string[] args)
        {
            // The path to the documents directory.
            string dataDir = Aspose.Imaging.Examples.Utils.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            //Load an existing image
            using (Aspose.Imaging.Image image = Aspose.Imaging.Image.Load(dataDir + "sample.bmp"))
            {
                //Create an instance of PsdOptions and set it’s various properties
                Aspose.Imaging.ImageOptions.PsdOptions psdOptions = new Aspose.Imaging.ImageOptions.PsdOptions();
                psdOptions.ColorMode = Aspose.Imaging.FileFormats.Psd.ColorModes.Rgb;
                psdOptions.CompressionMethod = Aspose.Imaging.FileFormats.Psd.CompressionMethod.Raw;
                psdOptions.Version = 4;

                //Save image to disk in PSD format
                image.Save(dataDir + "output.psd", psdOptions);

                // Display Status.
                System.Console.WriteLine("Export to PSD performed successfully.");
            }
        }
    }
}

File: Examples/VisualBasic/ModifyingAndConvertingImages/ExportImageToPSD.vb
Imports Aspose.Imaging.FileFormats.Psd
Imports Aspose.Imaging.ImageOptions

Namespace Aspose.Imaging.Examples.VisualBasic.ModifyingAndConvertingImages
    Public Class ExportImageToPSD
        Public Shared Sub Run()
            ' To get proper output please apply a valid Aspose.Imaging License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx.")

            ' ExStart:ExportImageToPSD
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_ModifyingAndConvertingImages()

            ' Load an existing image
            Using image__1 As Image = Image.Load(dataDir & Convert.ToString("sample.bmp"))
                ' Create an instance of PsdOptions and set it’s various properties
                Dim psdOptions As New PsdOptions()
                psdOptions.ColorMode = ColorModes.Rgb
                psdOptions.CompressionMethod = CompressionMethod.Raw
                psdOptions.Version = 4

                ' Save image to disk in PSD format
                image__1.Save(dataDir & Convert.ToString("ExportImageToPSD_output.psd"), psdOptions)

                ' Display Status.
                Console.WriteLine("Export to PSD performed successfully.")
            End Using
            ' ExEnd:ExportImageToPSD
        End Sub
    End Class
End Namespace